pkg/bff/gds.go

```diff
@@ -81,9 +81,9 @@ func (s *Server) Lookup(c *gin.Context) {
 	req := &gds.LookupRequest{Id: params.ID, CommonName: params.CommonName}
 
 	// Create an RPC func for making a parallel GDS request
-	lookup := func(ctx context.Context, client gds.TRISADirectoryClient, network string) (_ proto.Message, err error) {
+	lookup := func(ctx context.Context, client *GDSClient, network string) (_ proto.Message, err error) {
```

Comment:
For a `struct` to implement an interface, it needs to have all of the methods the interface defines. I didn't think you could embed an interface into a struct - but if that compiles, I'm not actually sure what that means. 

For the embedding, what I was hoping you could do was something like:

```go
type GDSClient struct {
    gds.tRISADirectoryClient
    members.tRISAMembersClient
}

client := &GDSClient{
    tRISADirectoryClient: gds.tRISADirectoryClient{cc: gdsConn},
    tRISAMembersClient: members.tRISAMembersClient{cc: membersConn},
}
```

What I didn't realize was that while the compiled Go code at `pkg/trisa/gds/v1beta1` exported an interface `TRISADirectoryClient` it doesn't [export the implementation](https://github.com/trisacrypto/trisa/blob/main/pkg/trisa/gds/api/v1beta1/api_grpc.pb.go#L34-L36) because of that lower case "t" … 

So unfortunately that means we'll have to do something like:

```go
type GDSClient struct {
    gds gds.TRISADirectoryClient
    members members.TRISAMEmbersClient
    gdsConn *grpc.ClientConn
    membersConn *grpc.MembersConn
}

func (c *GDSClient) Lookup(ctx context.Context, in *gds.LookupRequest, opts …grpc.CallOption) (*gds.LookupReply, error) {
    return c.gds.Lookup(ctx, in, opts...)
}

func (c *GDSClient) List(ctx context.Context, in *members.ListRequest, opts …grpc.CallOption) (*members.ListReply, error) {
    return c.members.List(ctx, in, opts...)
}
```

Etc. 

I still think it's useful to do this, if somewhat annoying.